zkevm-circuits/src/evm_circuit/witness.rs

```diff
@@ -459,6 +493,26 @@ impl Rw {
                 F::zero(),
                 F::zero(),
             ],
+            Self::TxAccessListAccountStorage {
+                rw_counter,
+                is_write,
+                tx_id,
+                address,
+                key,
+                value,
+                value_prev,
+            } => [
+                F::from(*rw_counter as u64),
+                F::from(*is_write as u64),
+                F::from(RwTableTag::TxAccessListAccountStorage as u64),
+                F::from(*tx_id as u64),
```

Comment:
it is not same with <https://hackmd.io/VCGteyZEQFKUjHA_A72f1Q>. Is this correct?